From f1f3842d8bc13c41e9fb099f238329e4b05e61fe Mon Sep 17 00:00:00 2001
From: Adam Jackson <ajax@redhat.com>
Date: Wed, 11 Sep 2013 18:19:15 -0400
Subject: [PATCH] xfree86: Bump to video ABI 15 and input ABI 20

Damage and mipointer stuff got rearranged in the last few dozen commits.

Signed-off-by: Adam Jackson <ajax@redhat.com>
---
 hw/xfree86/common/xf86Module.h | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/hw/xfree86/common/xf86Module.h b/hw/xfree86/common/xf86Module.h
index e0cec05..51645ea 100644
--- a/hw/xfree86/common/xf86Module.h
+++ b/hw/xfree86/common/xf86Module.h
@@ -80,8 +80,8 @@ typedef enum {
  * mask is 0xFFFF0000.
  */
 #define ABI_ANSIC_VERSION	SET_ABI_VERSION(0, 4)
-#define ABI_VIDEODRV_VERSION	SET_ABI_VERSION(14, 1)
-#define ABI_XINPUT_VERSION	SET_ABI_VERSION(19, 2)
+#define ABI_VIDEODRV_VERSION	SET_ABI_VERSION(15, 0)
+#define ABI_XINPUT_VERSION	SET_ABI_VERSION(20, 0)
 #define ABI_EXTENSION_VERSION	SET_ABI_VERSION(7, 0)
 #define ABI_FONT_VERSION	SET_ABI_VERSION(0, 6)
 
-- 
1.8.4.rc3

